src/main/test/java/org/linecode/server/business/UserServiceImplTest.java

```diff
@@ -0,0 +1,41 @@
+package org.linecode.server.business;
+
+import com.github.msteinbeck.sig4j.signal.Signal1;
+import junit.framework.TestCase;
+import org.junit.Test;
+import org.linecode.server.persistence.UserRepository;
+import org.mockito.Mockito;
+
+import static org.mockito.Mockito.when;
+
+public class UserServiceImplTest extends TestCase {
+
+    UserServiceImpl test = new UserServiceImpl(Mockito.mock(UserRepository.class),Mockito.mock(Signal1.class));
+
```

Comment:
Cambiare da formato JUnit3 a JUnit4 senza estendere il TestCase e con il `@Before`
```suggestion
public class UserServiceImplTest {

    private UserServiceImpl test;
    
     @Before
     public void setUp() {
         test = new UserServiceImpl(Mockito.mock(UserRepository.class), Mockito.mock(Signal1.class));
     }
```